metrics/cli/test/deploy/deployer_test.dart

```diff
@@ -236,8 +225,8 @@ void main() {
         await deployer.deploy();
 
         verifyInOrder([
-          firebaseCommand.login(),
-          firebaseCommand.addFirebase(any, any),
+          firebaseService.login(),
+          firebaseService.createWebApp(any),
```

Comment:
It seems like we should update the description of this test. Also, please review all descriptions of the changed tests.